src/IOManagement.cpp

```diff
@@ -0,0 +1,93 @@
+// combined source file of digitalPin.cpp and analogPins.cpp 
+// combined variables / functions given the name hybrid
```

Comment:
Usually, comments like these are better placed in the PR description than in the code, since it describes what you did and not what the code is accomplishing.